packages/datagrid/src/components/DefaultCellRenderer/DefaultCellRenderer.component.js

```diff
@@ -15,11 +15,12 @@ export default function DefaultCellRenderer({ avroRenderer, colDef, value, getCo
 	let content;
 
 	if (data.loading) {
-		content = <Skeleton />;
+		content = <Skeleton key="1" />;
 	} else {
 		content = [
-			<QualityIndicator qualityIndex={value.quality} />,
+			<QualityIndicator key="1" qualityIndex={value.quality} />,
```

Comment:
Please don't use the same key for different elements.
Here React reconciliation will consider that the skeleton is the same element as Quality indicator, and can do more extra checks to determine the changes in the DOM. Setting different keys will tell React that they are not the same elements, so it can drop the old one and create the new one.